app/src/components/main/analytics/events/pluginsPageEvents.js

```diff
@@ -14,21 +14,75 @@
  * limitations under the License.
  */
 
+export const PLUGINS_PAGE = 'plugins';
+
+export const getPluginFilterTabClickEvent = (tabKey) => ({
+  category: PLUGINS_PAGE,
+  action: `Click on tab ${tabKey} in Plugins`,
+  label: `Open tab ${tabKey} in Plugins`,
+});
+export const getPluginItemClickEvent = (pluginName) => ({
+  category: PLUGINS_PAGE,
+  action: `Click on ${pluginName} plugin in Plugins`,
+  label: `Open page with info for ${pluginName} plugin in Plugins`,
+});
+export const getDisablePluginItemClickEvent = (pluginName) => ({
+  category: PLUGINS_PAGE,
+  action: `Click on ${pluginName} plugin enabled switcher`,
+  label: `Disable ${pluginName} plugin`,
+});
+export const getUninstallPluginBtnClickEvent = (pluginName) => ({
+  category: PLUGINS_PAGE,
+  action: `Click on Btn uninstall plugin for ${pluginName}`,
+  label: `Arise Modal uninstall plugin for ${pluginName}`,
+});
+
 const PLUGINS_MODAL = 'Modal plugins';
 export const PLUGINS_PAGE_EVENTS = {
+  CLICK_UPLOAD_BTN: {
+    category: PLUGINS_MODAL,
+    action: 'Click on Bttn Upload',
```

Comment:
Please change `Bttn` to `Btn` to make everything consistent. Check other places too